diff --git a/node_modules/@emdgroup-liquid/liquid/dist/vue.js b/node_modules/@emdgroup-liquid/liquid/dist/vue.js
index 725d521..6f417c0 100644
--- a/node_modules/@emdgroup-liquid/liquid/dist/vue.js
+++ b/node_modules/@emdgroup-liquid/liquid/dist/vue.js
@@ -1,7 +1,7 @@
 /* eslint-disable */
 /* tslint:disable */
 /* auto-generated vue proxies */
-import { defineContainer } from './vue-component-lib/utils';
+import { defineContainer } from './vue-component-lib/utils.js';
 export const LdAccordion = /*@__PURE__*/ defineContainer('ld-accordion', undefined, [
     'brandColor',
     'detached',
